animation_nodes/data_structures/meshes/mesh_data.pyx

```diff
@@ -211,31 +240,30 @@ cdef class Mesh:
         Vertices: {len(self.vertices)}
         Edges: {len(self.edges)}
         Polygons: {len(self.polygons)}
-        UV Maps: {self.getUVMapNames()}
-        Vertex Colors: {self.getVertexColorLayerNames()}""")
+        UV Maps: {self.getAttributeNames("UV_MAP")}
+        Vertex Colors: {self.getAttributeNames("VERTEX_COLOR")}
+        Custom Attributes: {self.getAttributeNames("CUSTOM")}""")
 
     def replicateMeshProperties(self, Mesh source, long amount):
         for ((meshProperty, _), (sourceMeshProperty, _)) in zip(
                 self.getMeshProperties(), source.getMeshProperties()):
```

Comment:
Use attributes directly instead of `getMeshProperties`.

Reply:
Removed.